data-structures/stack.md

```diff
@@ -60,4 +62,23 @@ var Stack = function() {}
 ```
 
 ### Challenge / Check for Understanding
+
 Find a partner and show each other your own Stack class. Explain how the class you wrote works.
+
+Make a pair of two and ask the following questions to each-other.
+
+- What is difference between stack and simple array?
+- Which method does the stack follows?(LIFO)
+- Which methods are used in the stack data structure?
+- What is run time complexity of the stack?
+- How to make simple class as a stack class?
+- What is the meaning of '_'(*underscore*) in the begining of the variable name? Refer [Playing with data structures in Javascript — Stack](https://blog.cloudboost.io/playing-with-data-structures-in-javascript-stack-a55ebe50f29d)
```

Comment:
Space after underscore? (`'_'`)

Reply:
May I make space after underscore? I didn't understand what is the change.

Reply:
Very minor one. Give a space before `(`.